### ISSUE TITLE
GCP and GKE support would be nice

### ISSUE THREAD

**Issue by nikolaydubina** [cduggn/ccExplorer#329]

**Comment by cduggn** [comment]
yeah the intention was to widen the scope, I've been busy with work. I'll be adding some new features and cleaning up some of the code logic over the autumn/winter months , feel free to contribute.
